releases/release-1.13/release-notes-draft.md

```diff
@@ -422,4 +422,37 @@ SIG Windows focused on improving reliability for Windows and Kubernetes support
 - Handle Windows named pipes in host mounts. ([#69484](https://github.com/kubernetes/kubernetes/pull/69484), [@ddebroy](https://github.com/ddebroy))
 - Fixed inconsistency in windows kernel proxy when updating HNS policy. ([#68923](https://github.com/kubernetes/kubernetes/pull/68923), [@delulu](https://github.com/delulu))
 
-
+##  External Dependencies
+
+- Default etcd server remains at v3.2.24. ([#68318](https://github.com/kubernetes/kubernetes/pull/68318))
+- The list of validated docker versions was updated to 1.11.1, 1.12.1, 1.13.1, 17.03, 17.06, 17.09, 18.06. ([#68495](https://github.com/kubernetes/kubernetes/pull/68495))
+- The default Go version was updated to 1.11.2. ([#70665](https://github.com/kubernetes/kubernetes/pull/70665))
+- The minimum supported Go version was updated to 1.11.2 ([#69386](https://github.com/kubernetes/kubernetes/pull/69386))
+- CNI is unchanged from v1.10: v0.6.0 ([#51250](https://github.com/kubernetes/kubernetes/pull/51250))
+- CSI is unchanged from v1.11:  0.3.0 ([#64719](https://github.com/kubernetes/kubernetes/pull/64719))
```

Comment:
Should this line go away in light of the next line?

Reply:
Woops, sorry about that,  good catch! I forgot to delete this when writing the new note.